types/makeup-floating-label/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for makeup-floating-label 0.0
+// Project: https://github.com/makeup-js/makeup-floating-label#readme
+// Definitions by: Timur Manyanov <https://github.com/darkwebdev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.9
+
+export default class FloatingLabel {
```

Comment:
This is declared with `module.exports = /*...*/`, so it should be an `export =` declaration. To import it with a default import like you're currently doing, you'll need to enable `esModuleInterop` in your `tsconfig.json`.

Reply:
thanks, fixed with `export = ...` and `import ... = require...`, is it ok now? I'm not sure about the best practices nowadays...